packages/webpack/package.json

```diff
@@ -28,7 +28,7 @@
     "access": "public"
   },
   "dependencies": {
-    "axios": "^0.21.1",
+    "axios": "^1.6.0",
```

Comment:
```suggestion
    "axios": "^1.0.0",
```
Let's not require the latest latest version.

Reply:
Actually, considering merging it as-is because of the vulnerability it fixes -- not that AppSignal's servers will do anything with that data, but if it's something that users might not want us sending to begin with.